src/Popup.js

```diff
@@ -14,19 +14,28 @@ const Popup = ({show, onClose, onSubmit}) => {
     }, [value, onSubmit, onClose]);
 
     return(
-        <div className={'popup' + (show ? '' : ' hidden')} id="myPopUp" >
-            <div className='popupInner'>
-                <h1>Add an Idea</h1>
+        <div className={"popup" + (show ? "" : " hidden")} id="myPopUp" >
+            <div>
+                <h1>{formType ? "Add an Idea" : "Would you like to change your Idea?"}</h1>
                 <form onSubmit={handleSubmit}>
-                    <label>Enter Idea:</label><br/>
-                    <input type="text" value={value} onChange={handleChange} /><br/>
-                    <input className="submitButton" type="submit" value="Add"/>
+                    <div>
+                        <label>{formType ? "Enter Idea:" : "New Idea:"}</label>
+                    </div>
+                    <div>
+                        <input type="text" value={value} onChange={handleChange} />
+                    </div>
+                    <div>
+                        <input className="submitButton" type="submit" value="Add"/>
+                    </div>
                 </form>
-                <button onClick={onClose}>Exit</button>
+                    <button onClick={onClose}>Exit</button>
                 
```

Comment:
As I've stated already, you should probably pull this out of Popup and provide it in children. Then you'll probably want to abstract the form so you don't have two sets of the same form.